packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/useLocationDetailView.ts

```diff
@@ -55,6 +68,12 @@ export const DEFAULT_LIST_OPTIONS = {
   pageSize: DEFAULT_PAGE_SIZE,
 };
 
+const listLocationItemsAction = createEnhancedListHandler<
+  ListLocationItemsHandlerOptions,
+  LocationItemData,
+  LocationItemType
+>(listLocationItemsHandler);
```

Comment:
Shouldn't need to provide these types as generics, if there is an issue with `createEnhancedListHandler` not being able to infer these types from `listLocationItemsHandler` we should address that in the util